tsconfig.json

```diff
@@ -1,19 +1,11 @@
 {
+    "extends": "tsconfigs/browser-module",
     "compilerOptions": {
+        "esModuleInterop": false,
+        "strictFunctionTypes": false,
         "target": "ES5",
-        "noImplicitAny": true,
-        "sourceMap": true,
```

Comment:
[tsconfigs sets this](https://github.com/mightyiam/tsconfigs/blob/1be54f2c1b2384828ced56327f77419e8a2a37a9/_private/common.json#L10).